lib/msal-node/package.json

```
@@ -70,5 +70,8 @@
     "axios": "^0.21.1",
     "jsonwebtoken": "^8.5.1",
     "uuid": "^8.3.0"
+  },
+  "engines" : {
+    "node" : ">=10.0.0 <16.0.0"
```

If we just want to support LTS even release versions, it should be: 

```suggestion
    "node" : "10 || 12 || 14"
```